pycqed/measurement/multi_qubit_module.py

```diff
@@ -2255,7 +2255,11 @@ def measure_cphase(qbc, qbt, soft_sweep_params, cz_pulse_name,
             cal_points=cp, upload=False, prep_params=prep_params,
             max_flux_length=max_flux_length,
             num_cz_gates=num_cz_gates)
-
+    # compress 2D sweep
+    if kw.get('compress_2D_sweep', False):
+        sequences, hard_sweep_points, soft_sweep_points, cf = \
+            sequences[0].compress_2D_sweep(sequences,
+                                           kw.get("compression_seg_lim", 300))
```

Comment:
Do we really need 2 keyword args? Why not "if kw.get('compression_seg_lim', None) is not None"?

Reply:
I think this would be compressing two distinct ideas into only one variable. What if we want to compress but without segment limit? Sure we could put it very high but it would be a bit hacky in my opinion